sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/android/PackageName.kt

```diff
@@ -14,3 +14,10 @@ internal fun Project.packageName(): String {
     }
   throw IllegalStateException("No source sets available")
 }
+
+internal fun Project.sqliteVersion(): String? {
+  val androidExtension = extensions.getByType(BaseExtension::class.java)
+  val minSdk = androidExtension.defaultConfig.minSdk ?: return null
+  if (minSdk >= 30) return "sqlite:3.25"
+  return "sqlite:3.18"
```

Comment:
Are there only two of these because we don't support any other versions between the two?

Reply:
yep!

Reply:
Is this still true? Our documentation currently uses `3.24`.

![image](https://user-images.githubusercontent.com/2387680/154557689-4ad613c5-435e-4932-ad56-d9891b77e5b7.png)